examples/lock-app/lock-common/src/LockEndpoint.cpp

```diff
@@ -22,25 +22,28 @@
 using chip::to_underlying;
 using chip::app::DataModel::MakeNullable;
 
-bool LockEndpoint::Lock(const Optional<chip::ByteSpan> & pin, OperationErrorEnum & err, OperationSourceEnum opSource)
+bool LockEndpoint::Lock(const chip::app::CommandHandler * commandObj, const Optional<chip::ByteSpan> & pin,
+                        OperationErrorEnum & err, OperationSourceEnum opSource)
 {
-    return setLockState(DlLockState::kLocked, pin, err, opSource);
+    return setLockState(commandObj, DlLockState::kLocked, pin, err, opSource);
```

Comment:
@bzbarsky-apple Is Handle needed here? What needs to be kept alive during the async operation? If, when the lock (app) finishes its operation,, it can generate the async LockOperation event as long as it knows the nodeId and FabricId...?

Reply:
`fabricIdx` and `nodeId` are now passed instead of `commandObj` and example has been adapted to go async.